modules/release-notes/snip-bug-fix-rhidp-4732.adoc

```diff
@@ -0,0 +1,9 @@
+[id="bug-fix-rhidp-4732"]
+= The backstage instance always failed to start in version 5.1.0
```

Comment:
Why this bug fix is related to Backstage, however it should be RHDH. Is there any RHDH component that causing Backstage instance failure?

Reply:
> Why this bug fix is related to Backstage, however it should be RHDH. Is there any RHDH component that causing Backstage instance failure?

The failure was due to a broken rbac-backend plugin startup. This was fixed in [PR](https://github.com/janus-idp/backstage-showcase/pull/1864).